### ISSUE TITLE
Hardware and Software Specifications for this research

### ISSUE THREAD

**Issue by Lulo187** [bzhangGo/sltunet#8]
I appreciate the work you have done. Can you tell us about the hardware and software specifications used to carry out this research? Provide detailed requirements

I am very grateful also you want to share it.

Thank You

**Comment by bzhangGo** [comment]
Hey @Lulo187,

Re software,

As mentioned [here](https://github.com/bzhangGo/sltunet?tab=readme-ov-file#requirement), we used python 3.8 and tensorflow 1.15 (which was based on nvidia-tensorflow). I'm not sure about other details now, though.

Re hardware,

The experiments could be performed on a single RTX 3090 (24GB) with gradient accumulation (e.g., update_cycle=16).

**Comment by Lulo187** [comment]
Well, thank you for the information
